NEWS-v1.4-juliet-rose.md

```diff
@@ -154,6 +154,7 @@
 * Fixed issue where Open File dialog would fail to open files whose names were explicitly typed (#4059)
 * Fixed issue causing Project Sharing to fail to set Access Control Lists when using NFS v4 and `username@domain` security principals (Pro #2415)
 * Fixed issue where dialog boxes [e.g. Git commit, Installing Pro Database drivers] could fail to show output with Limit Console Output turned on (Pro #2284)
-* Fixed issue where uploading a .zip archive when unzip was not on PATH would cause a cryptic error. (#9151)
+* Fixed issue where uploading a .zip archive when unzip was not on PATH would cause a cryptic error (#9151)
+* Fixed issue where RStudio occasionally reset the locale to the "default" locale on Windows (#9350)
```

Comment:
Is this change intended for Juliet Rose?

Reply:
It is (and evidently I did not remember to update the base for this PR).

Should be properly targeted now.